internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -40,14 +40,16 @@ Now you can connect to your Atlas cluster with: mongosh -u %s -p %s %s
 
 `
 const quickstartTemplateCloseHandler = `
-You can connect to your Atlas cluster with the following user: 
+Enter 'atlas cluster watch %s' to learn when your cluster is available.
+`
+
+const quickstartTemplateStoreWarning = `
+Please store your database authentication access details in a secure location: 
```

Comment:
[q] Should we show the mongosh string to copy and paste into the terminal to connect to the cluster?

Reply:
Great point, we do display this at the end of quickstart if `skipMongosh` is not disabled, it was actually moved to the end as part of the improvements